packages/office-ui-fabric-react/src/components/List/List.Props.ts

```diff
@@ -41,6 +41,26 @@ export interface IListProps extends React.HTMLAttributes<List | HTMLDivElement>
   /** Optional callback to get the item key, to be used on render. */
   getKey?: (item: any, index?: number) => string;
 
+  /**
+   * Called by the list to get information about a page.
+   * Can be used if all information (item count, height) can be
+   * determined at once.
+   */
+  getPage?: (itemIndex?: number, visibleRect?: IRectangle) => {
```

Comment:
naming, maybe more like `getPageInformation`? or `getPageMetadata`? `getPage` sounds a bit like it would return the actual items